buddyboss/buddyboss-platform
# Group Message send multiple times when triggering the 'send message' button multiple time

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2437 
Fixes #2438

### How to test the changes in this Pull Request:

1. Go to Group Messages
2. Write a message
3. Click on the button multiple times.
4. See Emails.



### Other information:
